contrib/nginx/files/nginx

```diff
@@ -0,0 +1,7 @@
+# nginx service
+
+type            = process
+command         = /usr/bin/nginx -g 'daemon off;'
+depends-on      = local.target
+restart         = true
```

Comment:
```suggestion
```

default already

also, we could probably patch in readiness (like in pipewire readiness.patch)..